platform/mcu/stm32f4/flash_writer.c

```diff
@@ -8,6 +8,10 @@
 #define FLASH_PROGRAM_SIZE 0 // default: 8-bit
 #endif
 
+#define SECTOR_COUNT 24
```

Comment:
How do you get the sector_count value ?

Reply:
STM32F4 Reference Manual (another important document, separate from datasheet):

![Screenshot from 2019-08-16 13-58-56](https://user-images.githubusercontent.com/2419961/63166249-226b5980-c02e-11e9-8eb4-5716be7ad4d4.png)

It shows that the F4s with the most flash have sectors 0 to 23 -> 24 sectors.